templates/explore/repo_search.tmpl

```diff
@@ -36,7 +36,7 @@
 </div>
 {{if and .PageIsExploreRepositories .OnlyShowRelevant}}
 	<div class="ui message explore-relevancy-note">
-		<span data-tooltip-content="{{.locale.Tr "explore.relevant_repositories_tooltip"}}">{{.locale.Tr "explore.relevant_repositories" ((print $.Link "?only_show_relevant=0")|Escape) | Safe}}</span>
+		<span data-tooltip-content="{{.locale.Tr "explore.relevant_repositories_tooltip"}}">{{.locale.Tr "explore.relevant_repositories" ((printf "%s?only_show_relevant=0&sort=%s&q=%s&language=%s" $.Link $.SortType $.Keyword $.Language)|Escape) | Safe}}</span>
```

Comment:
But are these parameters always set?

Reply:
If you take a look at the cod e above, they all used there, so they are always set